stories/3-follow-the-color.md

```diff
@@ -0,0 +1,22 @@
+# Chore: Follow the color as your refactoring guide
+
+IDEs will give you tons of information via syntax highlighting. That information
+is in color.
+
+In our code we see a bouquet of color indicating:
+
+* keywords
+* instance variables
+* strings
+* escape keys inside strings
+
+This color can guide you towards cleaning up the code. In object-oriented
+languages like Ruby, everything should be about passing messages. Messages are
+your default IDE color. When you see other colors, you should be worried. In the
+case of instance variables and literals like string, it is an indication that
+your object oriented code is data or literal obsessed. More than likely the code
+is going to be having inappropriate non-message passing contact with those oddly
+colored items.
+
+In this story we are going to take a pass at stripping out or at least
```

Comment:
Is there some guidance we can provide about how far to go? I like that it's open ended to let people do their own thing, but I have a feeling I wouldn't go as far as intended. Would something like "no color in public methods" make sense?

Reply:
Hmm, I'd like this to be open ended. I have done this as a workshop and it has been pretty cool seeing where people go with this and then discussing. 

I'm not sure that no color in public methods is a good guideline, because I haven't tried it. It might be too far???

The interesting thing about this exercise vs the green field tic-tac-toe is that this is all about finding ambiguous limits in the work.

Reply:
Wondering if you can go through these exercises and we can see where it ends up?